flare/dft_interface/vasp_util.py

```diff
@@ -0,0 +1,87 @@
+from pymatgen.io.vasp.inputs import Poscar
+from pymatgen.io.vasp.outputs import Vasprun
+from subprocess import call
+import numpy as np
+from flare import struc
```

Comment:
I suggest keeping imports targeted to the individual object (as you did above for Pymatgen! :) ) so e.g. 
`from flare.struc import Structure`
Unless, did you do this to keep it clear the difference between a pymatgen structure and a flare structure?

Reply:
Changed to the above suggestion